main.go

```diff
@@ -92,6 +119,15 @@ func init() {
 	cli.BoolVar(&showHelp, "help", false, "display help and exit")
 	cli.BoolVar(&debug, "debug", false, "debug mode")
 	cli.StringVar(&semanticTheme, "theme", "green", "Semantic-ui theme to use")
+	cli.BoolVar(&enableNetworkSetup, "configure-network", false, "Configure networks automatically by subspace")
+	cli.BoolVar(&enableDnsmasq, "enable-dnsmasq", false, "Use dnsmasq")
+	cli.StringVar(&nameserver, "nameserver", "1.1.1.1", "DNS server used by clients")
+	cli.StringVar(&networkIPv4, "network-ipv4", "10.99.97.0/24", "IPV4 network address to create. First one is reserved by server.")
+	cli.StringVar(&networkIPv6, "network-ipv6", "fd00::10:97:0/64", "IPV6 network address to create. First one is reserved by server.")
+	cli.StringVar(&endpointHost, "endpoint-host", "", "WireGuard  device's endpoint hostname. By default, http-host is used.")
```

Comment:
you can set the default here for endpointHost to `httpHost` thus removing the need to check later on

Reply:
I think if I change this line like:

```
cli.StringVar(&endpointHost, "endpoint-host", httpHost, "WireGuard  device's endpoint hostname. By default, http-host is used.")
```

httpHost is empty string in this line, so default value of `endpointHost` also will be empty string.

I moved the check code from `handler.go` to `main.go to` make it clear that default value of `endpointHost` is `httpHost`.